samples/subsys/usb/hid-keyboard/boards/apollo4p_blue_kxr_evb.overlay

```diff
@@ -0,0 +1,17 @@
+#include <zephyr/dt-bindings/input/input-event-codes.h>

```

Comment:
Please add these to your board devicetree. My threshold for accepting overlays in samples is very high to keep the samples as clean as possible.

Reply:
Hi @jfischer-no, sorry for not knowing the rule for overlay. I have moved the configuration to board device tree. will take note the same for future PR.

Reply:
@zeonchew a developer's personal threshold is not a rule in Zephyr.

The only rule is if the device is on the board, then add it to the board and not a sample/test overlay.
If it is not on-chip or onboard an overlay should always be acceptable practice.

Reply:
Thanks @RichardSWheatley, Thanks for the explanation. 
In this case it is indeed better to put them into board as the declaration indeed belongs to the EVB boards. 
Thank you.

Reply:
> The only rule is if the device is on the board, then add it to the board and not a sample/test overlay. If it is not on-chip or onboard an overlay should always be acceptable practice.

If there are no buttons on the board, the overlays would not be acceptable in any case!